realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -221,63 +225,26 @@ public static SyncUser login(final SyncCredentials credentials, final String aut
      * @param credentials credentials to use.
      * @param authenticationUrl server that the user is authenticated against.
      * @param callback callback when login has completed or failed. The callback will always happen on the same thread
-     *                 as this this method is called on.
+     * as this this method is called on.
```

Comment:
The new auto-formatter formats it this way. We need to fix the formatter instead then, which is outside the scope of this PR.